pkg/tbtc/signing_test.go

```diff
@@ -0,0 +1,191 @@
+package tbtc
+
+import (
+	"context"
+	"fmt"
+	"github.com/keep-network/keep-core/pkg/chain"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+	"github.com/keep-network/keep-core/pkg/tecdsa"
+	"github.com/keep-network/keep-core/pkg/tecdsa/signing"
+	"math/big"
+	"reflect"
+	"testing"
+)
+
+func TestSigningRetryLoop(t *testing.T) {
+	chainConfig := &ChainConfig{
+		GroupSize:       10,
+		GroupQuorum:     8,
```

Comment:
Do we need to set the quorum for signing retry tests? I think it is only important for DKG.

Reply:
We don't: 62f829d7